packages/store/src/state-factory.ts

```diff
@@ -114,20 +119,45 @@ export class StateFactory {
     }
   }
 
+  connectActionHandlers(mappedStores: MappedStore[]): any {
+    if (this._connected) return;
+    this._actions
+      .pipe(
+        filter((ctx: ActionContext) => ctx.status === ActionStatus.Dispatched),
+        mergeMap(({ action }) => {
+          return this.invokeActions(this._actions, action).pipe(
+            map(() => {
+              return <ActionContext>{ action, status: ActionStatus.Completed };
+            }),
+            defaultIfEmpty(<ActionContext>{ action, status: ActionStatus.Canceled }),
+            catchError(err => {
+              return of(<ActionContext>{ action, status: ActionStatus.Errored });
+            })
+          );
+        })
+      )
+      .subscribe(ctx => {
+        this._actionResults.next(ctx);
+      });
+    this._connected = true;
+  }
+
+  private get rootStateOperations(): InternalStateOperations<any> {
```

Comment:
Getters/Setters should be before methods.

Reply:
Done. I moved it to the top.